### PR TITLE
[atip] Add new steps for new bdd feature

### PR DESCRIPTION
**Pull request by chenxix**

### CODE REVIEW

File: tools/atip/atip/web/steps.py [hunk 1]
```diff
@@ -179,3 +179,56 @@ def i_accept_alert(context):
...
+@step(u'I wait {n:d} seconds')
+def wait_senconds(context,n):
+   time.sleep(n)
+
+@step(u'I go to frame "{key}"')
+def i_visit_frame(context, key):
+    context.app.driver.switch_to_frame(key)
+    assert True
+
+@step(u'I jump frame')
```
Comment (Honry) [inline tools/atip/atip/web/steps.py]: Better rename as "I go out of frame"

File: tools/atip/atip/web/steps.py [hunk 2]
```diff
@@ -179,3 +179,56 @@ def i_accept_alert(context):
...
+def i_jump_frame(context):
+    context.app.driver.switch_to_default_content()
+    assert True
+
+@step(u'I should see nothing in "{key}"')
```
Comment (Honry) [inline tools/atip/atip/web/steps.py]: Please base on latest version, this step had been redefined by me.

**Comment by Honry** [comment]
I suggest you only submit the web.py first, and you can keep your personal steps at your suite, next step we plan to sort out the steps.py.

File: tools/atip/atip/web/web.py [hunk 3]
```diff
@@ -363,9 +396,13 @@ def press_element_by_key(self, key, display=True):
         element = self.__get_element_by_key(key, display)
         print "%s == %s" % (element.get_attribute("id"), element.get_attribute("class"))
         if element:
-            element.click()
-            return True
-
+            try:
+                element.click()
+                return True
+            except WebDriverException :
```
Comment (Honry) [inline tools/atip/atip/web/web.py]: This workaround at exception does not make sense, if this is a bug of xwalkdriver, please file a bug to track.

**Comment by cicili** [comment]
LGTM

**Pull request merged**